### PR TITLE
Move resources from /src/common/resources to /src/resources

### PR DESCRIPTION
**Pull request by stanleygomes**
# PR Details
Resources folder new location

## Description

Move resources from `/src/common/resources` to `/src/resources`

## Related Issue 

Issue Title ([143](https://github.com/SoftboxLab/juggernaut/issues/143))

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ x ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by stanleygomes** [comment]
Related to #143

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ These are some of patterns definitions to help us to keep a default configuratio
 ## :open_file_folder: Project structure
 
 - **src/assets**: Static files (images, fonts and icons);
+- **src/resources**: Api calls;
```
Comment (stanleygomes) [inline README.md:60]: Quem tiver uma sugestão melhor de descrição, só mandar ai, fiquei sem ideia

Comment (vilaboim) [inline README.md:60]:
```suggestion
- **src/resources**: API and endpoints configurations;
```

**Review by vilaboim — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stanleygomes) [reply README.md]: Great!

**Review by vilaboim — APPROVED** [review]

**Pull request merged**